packages/sign_in_with_apple/sign_in_with_apple/example/pubspec.yaml

```diff
@@ -5,7 +5,7 @@ version: 1.0.0+1
 
 environment:
   sdk: ^3.3.0
-  flutter: ">=3.19.1 <4.0.0"
+  flutter: ">=3.35.0 <4.0.0"
```

Comment:
Probably should bump this across the board now, so we don't need to special case the examples for CI. It's been out for long enough I suppose.